### ISSUE TITLE
Incorporate new updates to business/product strategy guidelines

### ISSUE THREAD

**Issue by timfrick** [w3c/sustyweb#124]
A list of eight suggested updates and potentially seven new guidelines have been added to the [business and product strategy doc](https://docs.google.com/document/d/1vbJ5t7hosd_Gl9eL0CO3hb_buNzgtKXjs2v_hozP-d8/edit?usp=sharing). Some of these overlap with existing guidelines, especially in UX design, as well. Also, some don't yet have success criteria or additional resources and none have recommendations for GRI impact.

However, they are good suggestions that impact business decision-making around sustainability for digital products and services, so they should be considered.

**Comment by AlexDawsonUK** [comment]
I'll go through each of the changes in turn as there is quite a lot of coverage (explaining reasoning behind the decisions):

- **Update 5:** Alterations have been made to the overview text, though not explicitly written as in the document as I wanted to keep the language in plain English and aligned with the rest of the spec. Tim has seen and approved the change.
- **Update 5.2:** The section has been updated as requested to include content on providing the tools they require to achieve the stated goals, and regarding the option a climate Working Group, plus a link to the Playbook.
- **Update 5.3:** A new SC has been created to include the additional content that didn't fit into the existing two SCs and all new thoughts have been integrated successfully, with a link to the Playbook and other relevant URLs - Note the two courses were not included due to the nature of their being paywalled and our focus being on open access.
- **Update 5.4:** A link to the Playbook is provided within resources (as in other guidelines for implementation advice).
- **Update 5.5:** Rather than add another paid tool, I've removed the existing one for neutrality as our focus is on availability for all and the preference should be on giving those with budgetary limits access to tooling.
- **Update 5.6:** Leveraging non-sustainability goals and metrics is already covered by existing guidelines and success criteria outside of the business section with enough scope not to require repeating. We have Web Accessibility (3.5), Security (3.16), Performance (2.26), Privacy by Design (5.20 & throughout), Usability (2.28), plus we have mapping overarching variables through guideline 2.1 and 3.1. I will add the links into relevant sections however.
- **Update 5.7:** The links have been added to the resources section for this guideline.
- **Update 5.12:** A link to the Playbook is provided within resources (as in other guidelines for implementation advice).

Potentially New Content:
- Quite a lot of the content rehashed what already existed within the WSGs, I've made notes in the doc striking through where duplication occurs and putting in bolded brackets where the content is suitably covered.
- Adjustments have been made as required to SCs for guidelines **2.5, 3.14**, and **5.5** to integrate the additional material that has been mentioned in order to strengthen the weight of the content. Guideline **5.11** has a new SC has there was enough grounds for a new SC on Failure Indicators (as we had no prior coverage on this area).

**Note:** All supplements including STAR have been updated to reflect these updates. They are already available live in the [living draft](https://w3c.github.io/sustyweb/drafts/) channel and will be released with the ninth draft when published.

**Comment by FrancoisBurra** [comment]
Fantastic work, @AlexDawsonUK @timfrick . Glad my feedback could help :)
